caffe2/ideep/utils/ideep_operator.h

```diff
@@ -111,4 +109,14 @@ class IDEEPOperator : public OperatorBase {
       : IDEEPOperator(operator_def, ws) {}                                     \
   virtual ~name() {}
 
+// Convert zero_point scales to min_max scales
+inline ideep::scale_t ConvertScales(
+    const std::vector<float> scales_z) {
```

Comment:
Const reference?

Reply:
Yes. Unchangeable. The return result gives conversion output.

Reply:
Sure. Will correct it.